### DISCUSSION TITLE
Can I see paper about LargeSPAdes?

### DISCUSSION THREAD

**Discussion by ods1108** [ablab/spades#1006] [Q&A]
Thank you for awesome program.

I'm interested about perfect hashing and memory management.
And I read paper about metaSPAdes and I read in supplementary that will be LargeSPAdes: scaling SPAdes assembler for large genomes.

The name of paper is changed?
Or can I see about that paper?

Thank you!

**Comment by asl** [comment]
The paper did not went out due to some reasons. However, all changes were integrated in the latest SPAdes releases. Consider using e.g. SPAdes 3.15.5

**Reply by ods1108** [reply]
Oh I see.

Thank you for answer!
